_docs/setup/kubernetes/ansible-install.md

```diff
@@ -101,12 +100,12 @@ ansible-playbook main.yml -e '{"cluster_flavour": "k8s", "cmd_path": "~/kubectl"
 
 - User wants to install Istio on Openshift with settings other than the default
 ```bash
-ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "jaeger": true, "delete_resources": true}}'
+ansible-playbook main.yml -e '{"istio": {"release_tag_name": "0.4.0", "auth": true, "delete_resources": true}}'
 ```
 
 - User wants to install Istio on Openshift but with custom add-on settings
 ```bash
-ansible-playbook main.yml -e '{"istio": {"delete_resources": true, "addon": ["grafana", "prometheus"]}}'
+ansible-playbook main.yml -e '{"istio": {"delete_resources": true, "addon": ["grafana", "prometheus", "jaeger"]}}'
```

Comment:
I saw somewhere prometheus was planned to be moved out of addons into the base install.  Might keep that in mind for the future (or I could be imagining that was planned).

Reply:
Very interesting to know! When/if that happens I'll update the playbook to install Prometheus in a similar fashion as Jaeger is installed